cfme/tests/control/test_bugs.py

```diff
@@ -624,3 +635,29 @@ def test_policy_condition_multiple_ors(
             assert state == "Policy simulation successful."
         else:
             assert state == "Policy simulation failed with: false"
+
+
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1740290])
+@pytest.mark.parametrize("page", ["ControlSimulation", "ControlImportExport", "ControlLog"],
+                         ids=["Simulation", "ImportExport", "Log"])
+def test_control_breadcrumbs(appliance, page, breadcrumb_locations):
+    """
+    Test to make sure breadcrumbs are visible and properly displayed
+
+    Bugzilla:
+        1740290
+
+    Polarion:
+        assignee: jdupuy
+        caseimportance: high
```

Comment:
**Optional:**
By default `caseimportance` is `high`.

Reply:
In my view it is better to be explicit here